build_tools/third_party/cmsis_device_f4/CMakeLists.txt

```diff
@@ -22,7 +22,7 @@ target_include_directories(cmsis_device_f407xx
   ${CMSIS_5_SOURCE_DIR}/CMSIS/Core/Include/
 )
 
-target_compile_definitions(cmsis_device_f407xx PUBLIC -DSTM32F407xx)
+target_compile_definitions(cmsis_device_f407xx PUBLIC -DSTM32F407xx HSI_VALUE=16000000 HSE_VALUE=8000000)
```

Comment:
Should `HSI_VALUE`  and/or `HSE_VALUE` be configurable? In that case we should set variables somewhere and use those here and in the other files.